lmms.txt

```diff
@@ -0,0 +1,12 @@
+[Section]
+Name = LMMS
+Version = 1.2.2
+License = GPL v2
+Description = LMMS is free and open source, cross-platform software for creating music, it is an alternative to programs like FL Studio
+Category = 1
+URLSite = https://lmms.io/
+URLDownload = https://github.com/LMMS/lmms/releases/download/v1.2.2/lmms-1.2.2-win32.exe
+SizeBytes = 31800000
```

Comment:
Since all the links that you added here are a fixed (and no floating versions, that would automatically change) please add SHA1 hashes also. This allows automatic detection of transmission failures, for example, or malign actions during the download.